lib/mycha_spec.coffee

```diff
@@ -0,0 +1,62 @@
+async = require 'async'
```

Comment:
Given that this is the main spec for this library, shouldn't it be in Cucumber?
Here is an example for how a description of a library API could look like in Cucumber: https://github.com/Originate/mortadella/blob/master/features/basic_usage.feature